docs/rbe-platforms.md

```diff
@@ -33,6 +33,10 @@ platform(
 
 Make sure to replace `gcr.io/YOUR:IMAGE` with your docker image url.
 
+Remote build actions will be run in your container via CMD, so you should make sure that
```

Comment:
nit: move this paragraph to a new section called "CMD and ENTRYPOINT" just before the next section ("Passing credentials for Docker images")